src/NWCClient.ts

```diff
@@ -456,7 +457,10 @@ export class NWCClient {
       };
 
       const onMessage = (message: {
-        data?: { type: "nwc:success" | unknown };
+        data?: {
+          type: "nwc:success" | unknown;
+          nostrWalletConnectUrl?: string;
```

Comment:
this needs to be changed to `relayUrl` and `walletPubkey` to be clear that the app secret is client-generated.